lib/redshift/client/connection.rb

```diff
@@ -7,7 +7,7 @@ class Connection
 
       def initialize(configuration)
         @original = PG.connect(configuration.params)
-        @original.type_map_for_results = PG::BasicTypeMapForResults.new(@original)
+        @original.type_map_for_results = extended_type_map_for(@original)
       end
 
```

Comment:
Pr looks fine, but you have no test added to cover your timezones issue. Could you add it?